src/FirebaseArray.js

```diff
@@ -314,7 +314,7 @@
             }
             rec.$id = $firebaseUtils.getKey(snap);
             rec.$priority = snap.getPriority();
-            $firebaseUtils.applyDefaults(rec, this.$$defaults);
+            angular.extend(rec, this.$$defaults);
```

Comment:
Defaults should only be applied if the key does not exist. This will always overwrite the existing value with the defaults, which isn't desirable.